modules/quadbin/bigquery/test/unit/quadbin.test.js

```diff
@@ -0,0 +1,82 @@
+// Quadbin encoding/decoding in JavaScript
+// Not used in the Analytics Toolbox.
+
+function tileToQuadbin (tile) {
```

Comment:
This sounds weird to me as the actual code is embedded in SQL, but it's ok to validate in some way the actual format.

Reply:
I included it in the PR, but it goes in deck.gl, so removing it from here.